.github/workflows/ci-cd.yml

```diff
@@ -153,3 +192,98 @@ jobs:
           use_qemu: 'true'
           registry_user: ${{ secrets.ECOSYSTEM_SERVICE_USER_DOCKER_HUB }}
           registry_token: ${{ secrets.ECOSYSTEM_ACCESS_TOKEN_DOCKER_HUB }}
+
+  create_prerelease:
+    permissions: write-all
+    runs-on: ubuntu-latest
+    needs: build_and_push
+    outputs:
+      RELEASE_ID: ${{ steps.prerelease.outputs.RELEASE_ID }}
+    env:
+      TAG: ${{ needs.build_and_push.outputs.TAG }}
+    steps:
+    - name: Create prerelease
+      id: prerelease
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        export RELEASE_ID=$( jq -r '.id' <<< $(
+        gh api \
+        --method POST \
+        -H "Accept: application/vnd.github+json" \
+        -H "X-GitHub-Api-Version: 2022-11-28" \
+        /repos/AxisCommunications/acap-runtime/releases \
```

Comment:
`AxisCommunication/acap-runtime`can be replaced by `${{ vars.GITHUB_REPOSITORY }}`